File: Beecrowd/Beginner/2003 - 2176/2060 - Bino's Challenge/Python/2060_BinosChallenge.py
numbers_in_list = int(input())

multiples_of_2 = multiples_of_3 = multiples_of_4 = multiples_of_5 = 0

numbers = list(map(int, input().split()))

for number in numbers:
    if number % 2 == 0:
        multiples_of_2 += 1
    if number % 3 == 0:
        multiples_of_3 += 1
    if number % 4 == 0:
        multiples_of_4 += 1
    if number % 5 == 0:
        multiples_of_5 += 1

print(f"{multiples_of_2} Multiplo(s) de 2")
print(f"{multiples_of_3} Multiplo(s) de 3")
print(f"{multiples_of_4} Multiplo(s) de 4")
print(f"{multiples_of_5} Multiplo(s) de 5")

File: Beecrowd/Beginner/2003 - 2176/2060 - Bino's Challenge/C++/2060_BinosChallenge.cpp
#include <bits/stdc++.h>

using namespace std;

int main() {
    int numbersInList;
    cin >> numbersInList;

    int multiplesOf2 = 0, multiplesOf3 = 0, multiplesOf4 = 0, multiplesOf5 = 0;

    while (numbersInList--) {
        int number;
        cin >> number;

        if (number % 2 == 0) multiplesOf2++;
        if (number % 3 == 0) multiplesOf3++;
        if (number % 4 == 0) multiplesOf4++;
        if (number % 5 == 0) multiplesOf5++;
    }

    cout << multiplesOf2 << " Multiplo(s) de 2" << endl;
    cout << multiplesOf3 << " Multiplo(s) de 3" << endl;
    cout << multiplesOf4 << " Multiplo(s) de 4" << endl;
    cout << multiplesOf5 << " Multiplo(s) de 5" << endl;

    return 0;
}